server/routes/referrals/serviceUserDetailsPresenter.test.ts

```diff
@@ -0,0 +1,80 @@
+import { ServiceUser } from '../../services/interventionsService'
+import ServiceUserDetailsPresenter from './serviceUserDetailsPresenter'
+
+describe(ServiceUserDetailsPresenter, () => {
+  const serviceUser = {
+    crn: 'X862134',
+    title: 'Mr',
+    firstName: 'Alex',
+    lastName: 'River',
+    dateOfBirth: '1980-01-01',
+    gender: 'Male',
+    ethnicity: 'British',
+    preferredLanguage: 'English',
+    religionOrBelief: 'Agnostic',
+    disabilities: ['Autism spectrum condition', 'sciatica'],
+  } as ServiceUser
```

Comment:
Why do we need the type assertion?

Reply:
Oops, left this in by accident, it's a good way of getting vs code to tell you the fields you're missing in a nicer way than the usual errors.